go/flags/endtoend/vtcombo.txt

```diff
@@ -173,7 +173,8 @@ Flags:
       --init-db-name-override string                                     (init parameter) override the name of the db used by vttablet. Without this flag, the db name defaults to vt_<keyspacename>
       --init-keyspace string                                             (init parameter) keyspace to use for this tablet
       --init-shard string                                                (init parameter) shard to use for this tablet
-      --init-tablet-type string                                          (init parameter) tablet type to use for this tablet. Valid values are: PRIMARY, REPLICA, SPARE, and RDONLY. The default is REPLICA.
+      --init-tablet-type string                                          (init parameter) tablet type to use for this tablet. Valid values are: REPLICA, RDONLY, and SPARE. The default is REPLICA.
+      --init-tablet-type-lookup                                          (Experimental, init parameter) if enabled, uses tablet alias to look up the tablet type from the existing topology record on restart and use that instead of init-tablet-type. This allows tablets to maintain their changed roles (e.g., RDONLY/DRAINED) across restarts. If disabled or if no topology record exists, init-tablet-type will be used.
```

Comment:
@sbaker617 I feel this comment should be updated so Vitess users understand this feature won't work on Vitess operator deployments, where tablet records are not left around in the topo

I think this current description may lead to those users trying this and believing there is a bug

Reply:
I'd rather not get into specifics like that. The operator's pruning behavior is configurable. There's a good chance that the replacement has a different alias (e.g. in another AZ) though too. But these are implementation details and examples of where the record would not be found and thus `init-tablet-type` is used.